src/components/Filters/types/Date.js

```diff
@@ -3,36 +3,41 @@ import { observer } from "mobx-react";
 import React from "react";
 import { DatePicker } from "../../Common/DatePicker/DatePicker";
 
+const userOffset = new Date().getTimezoneOffset();
+const convertDateToUTC = (date) => new Date(new Date(date).getTime() - userOffset * 60 * 1000);
+const convertUTCtoLocalDate = (utcDateString) => new Date(new Date(utcDateString).getTime() + userOffset * 60 * 1000);
```

Comment:
minor tweak just so we define milliseconds in an hour once and reuse it
```suggestion
const msInHour = 60 * 1000;
const convertDateToUTC = (date) => new Date(new Date(date).getTime() - userOffset * msInHour);
const convertUTCtoLocalDate = (utcDateString) => new Date(new Date(utcDateString).getTime() + userOffset * msInHour);
```